torchmd/run.py

```diff
@@ -277,7 +277,7 @@ def dynamics(args, mol, system, forces):
     for k in range(args.replicas):
         npy_name = os.path.join(args.log_dir, args.output + f"_{k}.npy")
         xyz_name = os.path.join(args.log_dir, args.output + f"_{k}.xyz")
-        converter_xyz_output(npy_name, xyz_name, mol.z)
```

Comment:
So here we are using `mol.z` and as you said are z coordinates for a moleculekit.Molecule.object, so it's going to fail

Reply:
I don't know who wrote that :/ it's obviously wrong

Reply:
I wrote that when I was using only npzMol class. Now that I'm using also moleculekit object, I know the error and I'm trying to fix it. Also the npzMol class is getting improved in this PR.